frontend/src/views/Settings.vue

```diff
@@ -61,6 +62,24 @@ export default {
   },
   computed: {
     ...mapState(["user", "loading"]),
+    name: () => name,
+  },
+  created() {
+    this.fetchData();
+  },
+  watch: {
+    $route: "fetchData",
+    reload: function (value) {
+      if (value === true) {
+        this.fetchData();
+      }
+    },
+  },
+  methods: {
+    ...mapMutations(["setLoading"]),
+    async fetchData() {
+      document.title = document.title + ` - ${this.name}`;
+    },
```

Comment:
Why is this added?

Reply:
The watch function

Reply:
Switching tabs in the settings also updates the title. If the watch-function is omitted, the instance-name only gets added to the initial load of the settings, but switching tabs removes it again